cli/src/templater.rs

```diff
@@ -594,6 +603,9 @@ where
     }
 }
 
+/// Template which, upon being forced (`extract()`ed), will evaluate its
+/// condition and select between a template for the true case and a template
+/// for the false case (which will yield nothing if it is [`None`]).
```

Comment:
This sounds more like a textual explanation of the code. Can you simplify? I think it'll be something like "Template which selects output based on a boolean condition."